examples/generated.yml

```diff
@@ -0,0 +1,28 @@
+api-keys:
```

Comment:
ive got that all in here: https://github.com/transcend-io/schema-sync/blob/main/examples/simple.yml

this file is an example of what gets spit out from `tr-pull` command. its not easy to add comments using yaml.dump. i also want to keep the docs concentrated in one yaml file that we can update as things change, instead of copy and pasting into each example file. i call out that `simple.yml` file in the docs here: https://github.com/transcend-io/schema-sync#transcendyml

Reply:
Okay cool, that's helpful!